release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -77,6 +77,11 @@ Because {product-title} {product-version} now uses a {op-system-base} 9.2 based
 
 User-defined tags for Microsoft Azure was previously introduced as Technology Preview in {product-title} 4.13 and is now generally available in {product-title} 4.14. For more information, see xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-user-defined-tags_installing-azure-customizations[Configuring the user-defined tags for Azure].
 
+[id="ocp-4-14-user-defined-tags-gcp"]
+==== User-defined tags for Google Cloud Platform (Technology Preview)
+
+You can now configure user-defined tags in Google Cloud Platform (GCP) for grouping resources and for managing resource access and cost. User-defined tags can be configured only in resources created by the {product-title} Image Registry Operator. For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-user-defined-labels-and-tags_installing-gcp-customizations[Managing the user-defined tags for GCP].
```

Comment:
- "You can now configure user-defined tags" -> "You can now configure user-defined **labels and** tags"? 
- Insert "User-defined labels can be configured..." before "User-defined tags can be configured..."?